LuaMenu/widgets/gui_battle_room_window.lua

```diff
@@ -144,6 +153,12 @@ local function SetupInfoButtonsPanel(leftInfo, rightInfo, battle, battleID, myUs
 		mapName = battle.mapName:gsub("_", " ")
 		mapName = StringUtilities.GetTruncatedStringWithDotDot(mapName, tbMapName.font, width - 22)
 		tbMapName:SetText(mapName)
+		local mapItem = WG.FeaturedMaps.Get(battle.mapName)
+		if mapItem ~= nil then
```

Comment:
This can just be "if mapItem then". Checking against nil sort of suggests that something special happens if it is false.

Reply:
Ah sure I'm happy to believe that is more "Luaonic", will update it.